src/main/app/claims/models/paymentDeclaration.ts

```diff
@@ -0,0 +1,15 @@
+export class PaymentDeclaration {
+
+  constructor (public paidDate: string, public explanation: string) {
+    this.paidDate = paidDate
```

Comment:
Remove these, public constructor args do the same already. Please read TypeScript doc.